### DISCUSSION TITLE
whats the best way to deploy to cloud

### DISCUSSION THREAD

**Discussion by gzhangx** [blitz-js/blitz#3055] [Q&A] [answered]
I am totally new to Blitz, trying to start a new project with it, but want to know before hand what's the best way to deploy this to cloud (for example, amazon). 

Should I:
1. deploy static website to S3 or azure static page, and deploy apis separately to lambdas etc
2. or should it deploy as a single package as a website.
3. or there are other ways to do it?

Thanks!
gz

**Comment by abuuzayr** [comment, marked as answer]
hey @gzhangx ! welcome to our community.

you can do both, but normally i believe we do (2).
For (1), if you deploy on Vercel, the lambdas are automatically deployed. But you will have to find somewhere else to host your database to connect your application to if that’s applicable.

You can find more deployment options in our docs: https://blitzjs.com/docs under Deploying to Production

Also if you’d like, we have a deployment channel over on our Discord: https://discord.blitzjs.com/ Come say hi!

**Comment by gzhangx** [comment]
Thank you!
